rustls/src/msgs/persist.rs

```diff
@@ -80,9 +81,11 @@ impl Codec for ClientSessionValue {
         self.server_cert_chain.encode(bytes);
     }
 
-    fn read(r: &mut Reader) -> Option<ClientSessionValue> {
+    pub fn read(r: &mut Reader, enabled_cipher_suites: &[&'static SupportedCipherSuite]) -> Option<ClientSessionValue> {
```

Comment:
The prototype of `Code::read` is too restrictive. We should eventually split `Codec` into two traits: one for reading and one for encoding.